docs/index.md

```diff
@@ -10,18 +10,166 @@ The Redpanda provider is designed for managing Redpanda clusters and Kafka resou
 
 It is also able to provide management of Kafka resources (topics, ACLs, and more) within Redpanda clusters deployed outside of Redpanda Cloud.
 
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- `client_id` (String, Sensitive) The id for the client. You need client_id AND client_secret to use this provider
+- `client_secret` (String, Sensitive) Redpanda client secret. You need client_id AND client_secret to use this provider
+- `cloud_provider` (String) Which supported cloud provider you are using (GCP, AWS). Can also be specified per resource
+- `region` (String) Cloud provider regions for the clusters you wish to build. Can also be specified per resource
+- `zones` (List of String) Cloud provider zones for the clusters you wish to build. Can also be specified per resource
```

Comment:
Do you think we should document this now? We don't yet support using these Redpanda attributes in each resource.

e.g https://github.com/redpanda-data/terraform-provider-redpanda/blob/main/redpanda/resources/network/resource_network.go#L181

Reply:
If we don't support them they should be removed from the schema. If we do, we should list em.

Reply:
Ok, so let's remove them from now since they are not supported in any of the resources.

Reply:
IMO that should be a follow up PR since this one is docs and that would be sorta functional. Thoughts?

Reply:
ok, so we should not document it if they don't work

https://github.com/redpanda-data/terraform-provider-redpanda/pull/48